scripts/qtcreator/templates/wizards/openframeworks/src/main.cpp

```diff
@@ -0,0 +1,13 @@
+#include "ofMain.h"
```

Comment:
@arturoc can't you reuse the existing templates in `scripts/templates`? This change introduces another set of template files, in a non-obvious location (i.e. not within `scripts/templates`), which is just confusing imo.